examples/data/wflow_upgrade/sbm/wflow_sbm_v0x.toml

```diff
@@ -190,7 +194,7 @@ parameter = "vertical.temperature"
 
 [[csv.column]]
 header = "Q"
-map = "gauges_grdc"
+map = "lateral.river.lake.locs"
 parameter = "lateral.river.q_av"
```

Comment:
We now have one case that is not tested anymore. (map name thit is not in input_options)
Could you switch to:

```toml
[[csv.column]]
header = "Q"
map = "gauges_grdc"
parameter = "lateral.river.q_av"

[[csv.column]]
header = "lake-volume"
map = "lateral.river.lake.locs"
parameter = "lateral.river.lake.storage"
```